newIDE/app/src/UI/ColorField/ColorPicker.js

```diff
@@ -76,11 +76,16 @@ class ColorPicker extends React.Component<Props, State> {
     this.setState({ displayColorPicker: false });
   };
 
+  getRgbBackground = (rgbColor: RGBColor) =>

```

Comment:
See my other comment - if you want to keep this in this file, move it outside the component, export it, rename it to something like "getRGBColorFromHex" and use it from the parent components when passing `color` to ColorPicker.

Reply:
I would actually need to convert hex to rgba with the current setup, if we dont want to alter colorpicker.js :) gd has some function to do that, but I'm not sure what its path is

Reply:
ok I implemented it in js, inside Utils/ColorTransformer.js